packages/styles/toast.css

```diff
@@ -1,13 +1,16 @@
 .Toast {
-  top: var(--top-bar-height);
-  position: fixed;
   color: #0b0e11;
   font-size: var(--text-size-small);
   z-index: var(--z-index-toast);
   align-items: center;
   box-sizing: border-box;
   display: flex;
   padding: 4px 15px;
+}
+
+.Toast.Toast--dismissible {
+  top: var(--top-bar-height);
+  position: fixed;
```

Comment:
This is a breaking change for a couple of reasons. A fixed toast is the standard implementation, while a non-dismissible toast is the variant. In order to make this not a breaking change, we should flip the implementation.

Maybe use `Toast--static`? I don't necessarily like describing the visual state of a component in its class, but I'm struggling to come up with a better option.

Reply:
would doing something like this work so I don't have to set the prop in one class and unset it in another?
```
.Toast:not(.Toast--static) {
  top: var(--top-bar-height);
  position: fixed;
}
```

Reply:
It would technically work, but it would also change specificity:

```css
.Toast { ... } /* specificity of [0, 1, 0] */
.Toast:not(.Toast--static) { ... } /* specificity of [0, 2, 0] */
```

Meaning that if someone was overriding cauldron's styles there's a possibility that we could be clobbering the styles that someone else added. If this were a new component it wouldn't necessarily be an issue but changing the specificity to an existing component could cause issues.

While specificity isn't implicitly called out in our breaking changes guidelines, we might need to consider documenting it. For now, we should follow the guiding principle:

> Before a breaking change is released, other alternatives should be considered